### PR TITLE
Fix issues when develop with visual studio on windows

### PR DESCRIPTION
**Pull request by python3kgae**
1. Initialize estimatedSimdLoopTripCount in ONNXReductionOpLowering::matchAndRewrite.
2. Add .vs to .gitignore

### CODE REVIEW

**Comment by tungld** [comment]
@jenkins-droid test this please

File: .gitignore [hunk 1]
```diff
@@ -187,3 +187,6 @@ dmypy.json
 # Visual Studio Code Files
 .vscode
 .devcontainer
+
+# Visual Studio Code Files
```
Comment (tungld) [inline .gitignore:191]: This line is duplicated. Could you remove it? Thank!

**Review by tungld — APPROVED** [review]
LGTM.

Just have a minor comment.

File: .gitignore (hunk 1 above)
Comment (python3kgae) [inline .gitignore:191]:
```suggestion
```

Comment (python3kgae) [reply .gitignore]: Fixed.

Comment (tungld) [reply .gitignore]: Thanks!

**Pull request merged**